File: back/Models/OrderHeader.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Polnet_api.Models
{
    public class OrderHeader
    {
        [Key]
        public Guid Id { get; set; }
        [MaxLength(200)]
        public int Sort { get; set; }
        public string Name { get; set; }

        [ForeignKey("OrderId")]
        public Order Order { get; set; }
        public Guid OrderId { get; set; }
    }
}

File: front/src/app/offers/model/order-header.ts
import { OrderElem } from './order-elem';

export interface OrderHeader {
    id?: string;
    sort: number;
    name: string;
    pricePurchase?: number;
    priceAfterDiscount?: number;
    priceBrutto?: number;
    orderId: string;
    orderElem?: OrderElem[];
    collapse?: true;
}
